public/app/core/services/NewFrontendAssetsChecker.ts

```diff
@@ -47,7 +48,7 @@ export class NewFrontendAssetsChecker {
       this.reloadIfUpdateDetected();
     }
     // Moving to dashboard (or changing dashboards)
-    else if (newLocationSegments[1] === 'd') {
+    else if (newLocationSegments[1] === 'd' && !playlistSrv.state.isPlaying) {
```

Comment:
maybe the "isPlaying" state of playListSrv should be in localstorage so it can survive a full page reload?

Reply:
Yes. I think that would be a good idea, but we would probably need to keep the full state of the `playlistSrv` in the localStorage, so we can restore it fully on a reload. We would have to be wary about making sure the state is properly purged when the user leaves the playlist.